docs/milestones.md

```diff
@@ -0,0 +1,21 @@
+# Milestones
+
+## [M0] - Modelo del Problema
+Entregable: Código que modele los elementos clave del problema financiero del estudiante, incluyendo la representación de los gastos fijos, gastos variables, y el presupuesto mensual del estudiante (relacionado con HU1 y HU2).  
+Viabilidad: Será viable cuando el sistema modele correctamente los gastos fijos, gastos variables y el presupuesto, permitiendo que el sistema pueda interactuar con estos elementos para reflejar cómo se distribuye el presupuesto del estudiante entre sus gastos. El sistema debe ser capaz de gestionar y mostrar correctamente los gastos fijos y variables, lo que es fundamental para las HUs 1 y 2, ya que ambas requieren un control preciso de los gastos dentro del presupuesto. La viabilidad será validada cuando las pruebas aseguren que, al introducir diferentes tipos de gastos, el sistema refleja correctamente el impacto de esos gastos en el presupuesto disponible.  
+
+## [M1] - Registro y Visualización de Gastos Fijos
+Entregable: Código que permita registrar y visualizar los gastos fijos del estudiante, como el alquiler e internet, con un resumen claro del total comprometido en estos gastos (vinculado a HU1).  
```

Comment:
¿Visualizar de nuevo?

Reply:
¿Y por qué se pide hacer el modelo de las HUs 1 y 2 si sólo se va a usar la primera en el segundo milestone?

Reply:
Porque es la base del problema. Por algún lugar tendré que empezar. Pongo entidades de las 4 HU y está mal. Lo rehago incluyendo 3 y son demasiados, cuando no quiero que se modelice la lógica, sino las entidades clave del desarrollo en los próximos milestones. Finalmente lo he dejado con 2 y tampoco está bien. No sé qué incluir en el M0, milestone interno. Creo que he entendido bien el guión, pero no consigo el aprobado.

Reply:
¿has entendido el concepto de producto mínimamente viable? Si estás modelando las HUs 1 y 2 pero sólo vas a usar la 1, ¿el producto es realmente mínimo?
En este objetivo lo imprescindible es que entendáis cómo se divide problema y como se planifica la solución del mismo. No que pongáis formalmente cosas que sean gramaticalmente correctas. En todo caso tenéis que veros a vosotros mismo, o a quien le toque desarrollarlo, leyendo esto y planteando qué tiene que hacer en ese instante y si lo que hace está bien.